src/components/MenuLink/MenuLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './MenuLink.scss';
+import { MenuNavLink } from '../MenuNavLink';
+
+const MenuLinks = [
+  { to: '/', text: 'home' },
+  { to: '/phones', text: 'phones' },
+  { to: '/tablets', text: 'tablets' },
+  { to: '/accessories', text: 'accessories' },
+];
+
+export const MenuLink = () => (
+  <nav>
+    <ul className="nav__list">
+      {MenuLinks.map(({ to, text }) => (
+        <MenuNavLink key={to} to={to} text={text} />
+      ))}
+    </ul>
+  </nav>
+);
```

Comment:
I think it's better not to map an array here. All these NavLinks have different 'to' and 'text', maybe we'll just code them here, without MenuNavLink component?